resolwe_bio/processes/support_processors/go_genesets.yml

```diff
@@ -0,0 +1,30 @@
+# ====================
+# Generate GO genesets
+# ====================
+---
+
+- slug: go-genesets
+  name: Generate GO genesets
```

Comment:
Geneset should be written with space: "gene set". Fix this everywhere. Here I suggest to change the name to "Import gene sets: GO terms".